pkg/hubble/metrics/api/context.go

```diff
@@ -304,35 +309,55 @@ func labelsContext(invertSourceDestination bool, wantedLabels labelsSet, flow *p
 			case "source_ip":
```

Comment:
i was thinking sometimes customers would want to see the IP if endpoints are explicitly ignored, but on second thought, i should ignore adding IPs too,